package.json

```diff
@@ -70,6 +70,7 @@
     "eslint-plugin-jasmine": "4.1.0",
     "eslint-plugin-jest": "23.8.2",
     "eslint-plugin-prettier": "3.1.2",
+    "eslint-plugin-react": "^7.22.0",
```

Comment:
Why do we need this? There is no React in this project.

Reply:
Dependency's removed after update of eslint config.